External/libgit2

```diff
@@ -1 +1 @@
-Subproject commit d5712ed2b33a18a4f9417d112bda7813c0570caa
```

Comment:
The full list of [changes to libgit2](https://github.com/libgit2/libgit2/compare/d5712ed2b33a18a4f9417d112bda7813c0570caa...bdf0e734506b5b18234d48a0e7c6995aeda30b9d)